Misc/NEWS.d/next/Library/2019-12-15-18-47-20.bpo-39040.tKa0Qs.rst

```diff
@@ -0,0 +1,2 @@
+Fix parsing of invalid Content-Disposition email headers by collapsing
+whitespace between encoded words in a bare-quote-string.
```

Comment:
This isn't specific to the content-disposition header, it applies to any mime header, since in theory any mime header can have paramaters.  So, 'Fix parsing of invalid mime header parameters by...'.